webview-dist/index.d.ts

```diff
@@ -1,12 +1,73 @@
-export interface QueryResult {
+interface QueryResult {
+    /** the number of rows affected by the query. */
     rowsAffected: number;
+    /**
```

Comment:
notice the impact of multi-line comments making their way into the `d.ts` file and in turn this will now show up in editors that support it.